src/patternfly/_themes.scss

```diff
@@ -0,0 +1,67 @@
+.pf-t-dark {
+  --pf-global--Color--100: var(--pf-global--Color--light-100);
+  --pf-global--Color--200: var(--pf-global--Color--light-200);
+  --pf-global--BorderColor: var(--pf-global--BorderColor--light);
+
+  .pf-c-button {
+    // btns
+    --pf-c-button--m-secondary--BoxShadowColor: var(--pf-global--Color--light-100);
+    --pf-c-button--m-secondary--Color: var(--pf-global--Color--light-100);
+    --pf-c-button--m-secondary--hover--BoxShadowColor: var(--pf-global--Color--light-100);
+    --pf-c-button--m-secondary--hover--Color: var(--pf-global--Color--light-100);
+    --pf-c-button--m-secondary--focus--BoxShadowColor: var(--pf-global--Color--light-100);
+    --pf-c-button--m-secondary--focus--Color: var(--pf-global--Color--light-100);
+    --pf-c-button--m-secondary--active--BoxShadowColor: var(--pf-global--Color--light-100);
+    --pf-c-button--m-secondary--active--Color: var(--pf-global--Color--light-100);
+
+    // tertiary inverse btn default styles
+    --pf-c-button--m-tertiary--BoxShadowColor: var(--pf-global--Color--light-100);
+    --pf-c-button--m-tertiary--Color: var(--pf-global--Color--light-100);
+    --pf-c-button--m-tertiary--hover--BoxShadowColor: var(--pf-global--Color--light-100);
+    --pf-c-button--m-tertiary--hover--Color: var(--pf-global--Color--light-100);
+    --pf-c-button--m-tertiary--focus--BoxShadowColor: var(--pf-global--Color--light-100);
+    --pf-c-button--m-tertiary--focus--Color: var(--pf-global--Color--light-100);
+    --pf-c-button--m-tertiary--active--BoxShadowColor: var(--pf-global--Color--light-100);
+    --pf-c-button--m-tertiary--active--Color: var(--pf-global--Color--light-100);
+  }
+
+  &.pf-m-transparent {
+    background-color: transparent;
+  }
+
+  &.pf-m-transparent-100 {
+    background-color: rgba($pf-color-black-1000, .42);
```

Comment:
should these be using custom properties? It seems kind of odd to use custom properties for some values, and css selectors/specificity for others.

Reply:
We are unsure if we are ever going to use these backgrounds, so for now I'll leave them as is. Also it's too ling to write the why, let's meet and talk.